### DISCUSSION TITLE
reac-hook17.0.1  mobx5.15  Uncaught (in promise) RangeError: Maximum call stack size exceeded     at initializeInstance

### DISCUSSION THREAD

**Discussion by wgh1997** [mobxjs/mobx#3296] [Q&A]
有的时候报错有的时候不报错 偶先
<img width="461" alt="image" src="https://user-images.githubusercontent.com/47033363/153824608-cf1c5611-f4a9-489e-bcb3-b6af47a2ab6e.png">
<img width="869" alt="image" src="https://user-images.githubusercontent.com/47033363/153824672-a9f93996-db20-427a-b1bc-4b1b6b2785f6.png">
<img width="969" alt="image" src="https://user-images.githubusercontent.com/47033363/153824851-8512355b-af59-4546-b78c-edfdba4a30a0.png">

**Comment by kubk** [comment]
Please provide a minimal reproduction: https://codesandbox.io/s/minimal-mobx-react-project-ppgml?file=/index.js
